spec/latest/json-ld-api/index.html

```diff
@@ -504,7 +513,7 @@
     </pre>
 
     <p>Note how in the output above all properties of a <a>node</a> are collected in a
-      single <a>JSON object</a> and how the <a>blank node</a> representing
+      single <a class="changed">dictionary</a> and how the <a>blank node</a> representing
```

Comment:
Here when we say: "Note how in the output above...", we're conflating the output of the algorithm (which is in "internal representation" form) with what's actually shown above, which is the JSON-LD serialization of that internal representation.

It seems like we need some common phraseology when talking about JSON-LD documents vs. internal representation and the examples of inputs/outputs. The examples are all going to be JSON-LD documents -- and so if we just jump into internal representation terminology to talk about them that may be confusing. Similarly, if we talk about the actual outputs of algorithms loosely without distinguishing between whether we're talking about the internal representation or the JSON-LD document that is a result of serializing it, we may cause confusion.

Perhaps we can just sprinkle in something like:

"The example above is the JSON-LD serialization of the output of the flattening algorithm. Note how ... JSON object ..."

Here we'd use JSON object because we're talking about the serialized output. Alternatively, we could go with:

"The example above is the JSON-LD serialization of the output of the flattening algorithm. Note how the algorithm's output ... dictionary ..."

Where we'd use "dictionary" because we're talking about the output itself. I think we may want to add these sorts of phrases where examples collide with talking about algorithm output in the spec.